preload_data.sh

```diff
@@ -1923,6 +1926,24 @@ EOF
   success "Cert manager is ready, preload can proceed."
 }
 
+#
+# check yq version
+# update it if not in the correct version
+#
+function check_yq() {
+  yq_version=$("${YQ}" --version | awk '{print $NF}' | sed 's/^v//')
+  yq_minimun_version=4.18.1
+
+  if [ "$(printf '%s\n' "$yq_minimun_version" "$yq_version" | sort -V | head -n1)" = "$yq_minimun_version" ]; then 
+    msg "YQ version is greater than or equal to ${yq_minimun_version}"
+  else
+    msg "YQ version is less than ${$yq_minimun_version}, updating yq"
+    wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/local/bin/yq
+    sudo chmod +x /usr/local/bin/yq
+    YQ=yq
+  fi
+}
```

Comment:
Hi @qpdpQ 

Could we just raise the error message if the yq version does not meet the criteria. Here is an [example](https://github.com/IBM/ibm-common-service-operator/blob/8ca0a3906e9d88670f7fb0936efc856a099539a6/convert_to_multi_instance.sh#L65).
The main considers are
1. The customer local machine could be amd, power or Z platform.
2. Downloading is not available for Airgap Env.